src/css/_footer.scss

```diff
@@ -35,11 +35,27 @@
   height: 28px;
 }
 
-@media screen and (max-width: 825px) {
-  .site-footer {
+@media only screen and (max-width: 767px),
+only screen and (min-width: 768px) and (max-width: 1023px) {
+
+   .site-footer {
+    position: static;
+    height: auto;
+    padding: 20px;
+    text-align: center;
+  }
+  .footer-left {
+    margin-left: 0;
+    margin-bottom: 0px;
+  }
+  .footer-social li {
+    margin-right: 15px;
     display: none;
   }
-  .footer-social {
+  .footer-left {
     display: none;
   }
```

Comment:
@Moeez905 should the first `.footer-left` block be removed? If it's not going to be displayed (per the second one) is there any point setting the margins to zero? For `.footer-social li` same'ish thing, if it isn't displayed is it worth setting margin? (Maybe that second one means it's effectively a spacer?)

Reply:
I'm unable to launch website on my local server by the regular method of "npm run start", this shows up "ERROR: "start:hugo" exited with 4294967295."
Meanwhile Yes! 
-from **.footer-social li** only the  " margin-right: 15px;"   property can be removed as it's not going to be displayed after all.
-for **.footer-left** I have to check it first on my server as simply removing it might cause slight misalignment

Reply:
I've been using docker for testing, seems to work okay: https://github.com/zaproxy/zaproxy-website#docker

Reply:
Did you `npm install`?

Reply:
yes it was already installed but trying again now

Reply:
removed dist too. still failed to launch with "ERROR: "start:webpack" exited with 1." shall I try uninstalling and re-installing node?

Reply:
+1 note that it now requires Node.js 20.

Reply:
I have things running locally. I can make adjustments/provide screenshots, if there's a few simple steps you can provide?

Reply:
@kingthorin I just wanted to check if removing:

.footer-left {
    margin-left: 0;
    margin-bottom: 0px;
  }
  
  makes any difference or otherwise it can be removed

Reply:
Seems good to me.

<details>
<summary>Normal desktop, doesn't get those due to screen size</summary>

![image](https://github.com/zaproxy/zaproxy-website/assets/7570458/7928e5fd-a1bc-42a6-9eff-019c2a896b53)

</details>

<details>
<summary>Small screen gets them, before removal</summary>

![image](https://github.com/zaproxy/zaproxy-website/assets/7570458/a21ef74a-a818-4830-ae70-0b2ea416b194)

</details>


<details>
<summary>Small screen gets them, but I removed them (seems fine to me)</summary>

![image](https://github.com/zaproxy/zaproxy-website/assets/7570458/4e1a7a09-307d-41ca-a83a-231be5cee5db)

</details>

Reply:
![Screenshot 2023-07-06 205141](https://github.com/zaproxy/zaproxy-website/assets/118293110/3b9f0e73-8a37-4d27-95b4-6b2a15e0581c)

My bad! it's still not working with the updates too

Reply:
> 

thanks !! its fine then